server/src/main/java/org/elasticsearch/Version.java

```diff
@@ -96,15 +96,15 @@
     public static final Version V_6_7_2 = new Version(6070299, org.apache.lucene.util.Version.LUCENE_7_7_0);
     public static final Version V_6_8_0 = new Version(6080099, org.apache.lucene.util.Version.LUCENE_7_7_0);
     public static final Version V_6_8_1 = new Version(6080199, org.apache.lucene.util.Version.LUCENE_7_7_0);
-    public static final Version V_6_8_2 = new Version(6080299, org.apache.lucene.util.Version.LUCENE_7_7_0);
```

Comment:
Was it added by mistake ?

Reply:
yes, that was a mistake. thanks! 
fixed on https://github.com/elastic/elasticsearch/pull/43583/commits/42c87991f374bcb0f5d8e8135650e2125b43561b